src/lib.rs

```diff
@@ -218,6 +225,11 @@ impl<IO: DfuIo> DfuSansIo<IO> {
         Ok(())
     }
 
+    /// Set the address onto which to download the firmware
```

Comment:
(just for consistency)
```suggestion
    /// Set the address onto which to download the firmware.
```